src/main/charts/confluence/values.yaml

```diff
@@ -183,6 +183,18 @@ synchrony:
     periodSeconds: 1
     # -- The number of consecutive failures of the Synchrony container readiness probe before the pod fails readiness checks
     failureThreshold: 30
+  resources:
+    jvm:
+      # -- The maximum amount of heap memory that will be used by the Synchrony JVM
+      maxHeap: "1g"
+      # -- The minimum amount of heap memory that will be used by the Synchrony JVM
+      minHeap: "1g"
+      # -- The memory allocated for the Synchrony stack
```

Comment:
Don't we need to define reserveCodeCache? The default for this in our Docker Releases is 256m

Reply:
Confuence itself has a `reserverCodeCache` of 256m not sure about synchrony.

Reply:
We don't set it the Quickstarts, nor in the managed Synchrony, so I don't think it's necessary. @badgersow may know more though.

Reply:
We don't use it in managed Synchrony, and I'm for keeping all deployment parameters consistent